refinery_core/Cargo.toml

```diff
@@ -26,7 +26,7 @@ toml = "0.5"
 url = "2.0"
 walkdir = "2.3.1"
 
-rusqlite = {version = "0.23", optional = true}
+rusqlite = {version = ">= 0.23, < 1", optional = true}
```

Comment:
yeah, i'd rather be conservative and limit to `0.25` even if rusqlite hasn't mostly changed it's api, we're dealing with minor versions

Reply:
Sure, I understand the concern. Changed to `0.25` :+1: